README.md

```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install django==1.7
```

Comment:
still confuse about django version
why not > 1.7
Any reason we need to fix the version?